### ISSUE TITLE
Error in online test bank question re Trusted Advisor?

### ISSUE THREAD

**Issue by fugi303** [awsccp/awsccp.github.io#46]
Full access to Trusted Advisor checks only comes with Business and Enterprise, no?

![image](https://user-images.githubusercontent.com/116797194/200323329-9ede951f-55bb-45e6-b11c-0e1207fd04a9.png)

![image](https://user-images.githubusercontent.com/116797194/200324577-1dcaa344-10ba-4e0e-8747-709ca90eeb10.png)
 
Also, I would say the question should be 'AWS customers get **full** access to....', since there is limited access with free and developer support.

**Comment by notwatermango** [comment]
Possible duplicate [issue](https://github.com/awsccp/awsccp.github.io/issues/9)

**Comment by dbclinton** [comment]
You're correct @notwatermango: this is something that I believe has changed a number of times over the past couple of years. For one thing, I don't believe the "Enterprise On-Ramp" support category even existed a year ago. 
For context, see: https://aws.amazon.com/premiumsupport/plans/?nc=sn&loc=1
Since we're most likely going to be publishing a new edition of the book within the next year, this issue probably won't be updated before then.
